src/app/ubs/ubs-admin/components/ubs-admin-notification/ubs-admin-notification.component.scss

```diff
@@ -1,3 +1,9 @@
+::ng-deep .notification-settings-matDialog {
```

Comment:
please use :host with ::ng-deep to save the style installation. If you need to change styles globally, write them in styles.scss file.